server/server.go

```diff
@@ -137,6 +138,10 @@ func (g *ReleaseManager) CheckForUpdate(p *Params, isLantern bool) (res *Result,
 	if err != nil {
 		return nil, fmt.Errorf("Bad app version string %v: %v", p.AppVersion, err)
 	}
+	if p.OS == OS.Android && p.AppVersion == "9999.99.99-dev" {
```

Comment:
Why limit this to Android?  I realize that we are currently experiencing the issue on Android, but this same thing should apply to all platforms, right?  Or were you envisioning this as a temporary thing, which we remove when the wayward clients are updated?

Reply:
Yeah, I was thinking of this more as a temporary measure, but as @myleshorton pointed out, these clients aren't even hitting here, so I'm going to close this.